creusot-contracts/src/lib.rs

```diff
@@ -439,9 +446,12 @@ mod base_prelude {
     };
 
     #[cfg(creusot)]
-    pub use crate::std::mem::size_of_logic;
-    #[cfg(creusot)]
-    pub use crate::util::such_that;
+    pub use crate::{
+        invariant::inv,
+        logic::such_that,
+        resolve::{resolve, structural_resolve},
+        std::mem::size_of_logic,
```

Comment:
I wonder if we could make this list shorter. There are certainly things here that we don't use so often.

I'm thinking of (the list is open to discussion):
- `such_that`
- `size_of_logic`
- `structural_resolve`
- `FnGhost`
- `InhabitedInvariant`
- `Subset`
- `OrdLogic`
- `Seq`
- `WellFounded`
- `Clone` (we no longer have our own)
- `Default` (we no longer have our own)
- `PartialEq` (we no longer have our own)
- `PartialOrd` (we no longer have our own)

Reply:
Of course, some of these are traits used in the `pearlite!` macro, so that it should still be exported, but with `as _`.

Reply:
I think it's worth keeping `OrdLogic` and `Seq` to mirror `PartialOrd` and `Vec` being in `::std::prelude`.